Libraries/test/TestWebApp/TestWebApp.csproj

```diff
@@ -20,7 +20,9 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.1.5" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.1.5"><PrivateAssets Condition="'%(PackageReference.Version)' == ''">all</PrivateAssets>
```

Comment:
What is the context of this change?

Reply:
Hm. Looks like something VS stuck in there. I'll revert this.